python/social_bot/envs/play_ground.py

```diff
@@ -93,6 +95,14 @@ def __init__(self,
             world_name (string): Select the world file, e.g., empty.world, play_ground.world,
                 grocery_ground.world
             tasks (list): a list of teacher.Task, e.g., GoalTask, KickingBallTask
+            goal_conditioned (bool): Turn on goal conditioned tasks.  Currently only GoalTask
```

Comment:
What is purpose of this option? As I know, for goal task currently, full state observation already includes desired goal and agent's current position(achieved_goal).

Reply:
This is mainly to return OrderedDict observation instead of flat array observation, as alf's hindsight relabeling depends on this format.  Modified comment to make it more clear.